src/services/permissions.js

```diff
@@ -88,7 +88,11 @@ module.exports = class PermissionsHelper {
 				})
 			}
 		} catch (error) {
-			throw error
+			return responses.failureResponse({
```

Comment:
@adithyadinesh0412 what if DB is down, will you still give bad request?

Reply:
We should have different response based on actual issue, bad request only if frontend is giving invalid data

Reply:
Resolved this. 
sending back error response if permission is not found.
For any other issues , catch block with send out generic error "Oops something went wrong" with 500 status code